specification/ai/Azure.AI.Agents/run_steps/models.tsp

```diff
@@ -273,6 +273,17 @@ model RunStepDeltaOpenAPIObject extends RunStepDeltaDetail {
   toolCalls?: RunStepDeltaOpenAPIToolCall[];
 }
 
+/** Represents the invocation of connected agent as a part of a streaming run step. */
+@added(Versions.v2025_05_15_preview)
+model RunStepDeltaConnectedAgentToolCall extends RunStepDeltaToolCall {
```

Comment:
is connected agent preview tool or also in stable release?

Reply:
I think, it is present in the stable version, but I think we are only allowed to modify the Versions.v2025_05_15_preview.

Reply:
But even in this case, the service team may want this to be in preview for a few weeks, before we add it to the stable "v1" version. Please consult with them on a case-by-case basis.